screens/Indoor/IndoorMapView.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from "react";
+import { View, ScrollView, AsyncStorage } from "react-native";
+import { JMSBFloor1 } from "../../assets/floormaps/mb/JMSBFloor1";
+import { HallFloor9 } from "../../assets/floormaps/hall/HallFloor9";
+import { JMSBFloor2 } from "../../assets/floormaps/mb/JMSBFloor2";
+import { HallFloorX } from "../../assets/floormaps/hall/HallFloorX";
+import { JMSBFloorX } from "../../assets/floormaps/mb/JMSBFloorX";
+
+function IndoorMapView() {
+    const [selectedBuilding, setSelectedBuilding] = React.useState("");
+    const [selectedFloor, setSelectedFloor] = React.useState("");
+
+    const buildingSelected = async () => {
```

Comment:
it was already mentioned in another comment I think, but using similar variable names "buildingSelected" and "selectedBuilding" might be a bit confusing. Better rename one of them.